packages/frameworks-solid-start/tsconfig.m.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "allowSyntheticDefaultImports": true,
+    "target": "esnext",
+    "newLine": "LF",
```

Comment:
```suggestion
```
Stylistic? Long term I want to line up all packages' output to be the same, and all code should be formatted the same: https://github.com/nextauthjs/next-auth/pull/6111#issue-1501676323